### DISCUSSION TITLE
How to display Pytest Fixtures in Timeline?

### DISCUSSION THREAD

**Discussion by garth-gross** [allure-framework/allure2#2230] [Questions & Support] [answered]
Is there a way to ensure that fixture information from a Pytest test run is factored is included on the Timelines section of an Allure report?

It looks like [this was possible at one point,](https://devcodef1.com/news/1026523/getting-allure-report-to-display-fixtures), but I am not seeing anything in the Allure2 code to denote the `timeline` or `fixtures` plugins exist.

**Comment by baev** [comment, marked as answer]
The article you provided looks like some AI-generated content. As far as I know, there is no such feature available.
